src/build/babel-transform.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+import {Transform} from 'stream';
+import * as logging from 'plylog';
+
+import File = require('vinyl');
+
+const requireg = require('requireg');
+const logger = logging.getLogger('js.compile');
+
+type FileCB = (error?: any, file?: File) => void;
+
+export class BabelTransform extends Transform {
+
+  _babel: any;
+  _preset: any;
+
+  constructor() {
+    super({objectMode: true});
+    try {
+      this._babel = requireg('babel-core');
+    } catch (e) {
+      logger.error(`${e.message}\nDid you install babel-core? Run \`npm i -g babel-core\``);
+    }
+    try {
+      this._preset = requireg('babel-preset-es2015');
```

Comment:
FYI: there is a [bug](https://github.com/web-animations/web-animations-next/issues/402) in web-animations-next-lite.min.js used by `neon-animated-page`. It becomes broken in strict mode   introduced by `babel-preset-es2015`. I ended up with using [babel-preset-es2015-without-strict](https://github.com/fancyboynet/babel-preset-es2015-without-strict) instead.